testsuite/unittests/test_iaf_psc_exp_ps_lossless.sli

```diff
@@ -263,13 +263,13 @@ dc_gen nrn_spike 1. 1. Connect
 
 %read out spike response of nrns:
 
-/spike_detector Create /det_nospike Set
+/spike_recorder Create /det_nospike Set
 nrn_nospike det_nospike Connect
```

Comment:
There might be more instances of the variable below, lets see if the test fails..
```suggestion
/spike_recorder Create /sr_nospike Set
nrn_nospike sr_nospike Connect
```